shared/featureconfig/config.go

```diff
@@ -20,12 +20,11 @@ The process for implementing new features using this package is as follows:
 package featureconfig
 
 import (
-	"sync"
-
 	"github.com/prysmaticlabs/prysm/shared/cmd"
 	"github.com/prysmaticlabs/prysm/shared/params"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
+	"sync"
```

Comment:
Can you group standard library to the top?

Reply:
ok, but i didnt touch anything there, it might be gofmt